docs/developer_guides/viewer/viewer_control.md

```diff
@@ -0,0 +1,80 @@
+# Python Viewer Control
+
+Similar to `ViewerElements`, Nerfstudio includes supports a pythonic interface to the viewer through which you can:
+
+* Set viewer camera pose and FOV
+* Set viewer scene crop
+* Retrieve the current viewer camera matrix
+* Install listeners for click events inside the viewer window
+
+## Usage
+
+First, instantiate a `ViewerControl` object as a class variable inside a model file.
+Just like `ViewerElements`, you can create an instance inside any class which inherits from `nn.Module`
+and is contained within the `Pipeline` object (for example the `Model`)
+
+```python
+from nerfstudio.viewer.server.viewer_elements import ViewerControl
+
+class MyModel(nn.Module):#must inherit from nn.Module
+    def __init__(self):
+        # Must be a class variable
```

Comment:
capitalization could be made consistent (personally I prefer capitalized, but not sure what repo mostly has)